src/main/java/com/squareup/javapoet/CodeBlock.java

```diff
@@ -269,6 +278,13 @@ public Builder addStatement(String format, Object... args) {
       return this;
     }
 
+    public Builder addStatement(CodeBlock codeBlock) {
```

Comment:
In that case I usually just put the `';\n'` myself instead of using addStatement.